### PR TITLE
feat: added backend .NET scaffolder plugin

### PR DESCRIPTION
**Pull request by alefcarlos**
Signed-off-by: Alef <alef.carlos@gmail.com>

## .NET scaffolder plugin

it adds `@plusultra/plugin-scaffolder-dotnet-backend` to Marketplace

### CODE REVIEW

**Comment by alefcarlos** [comment]
@awanlin I created another PR and I'm gonna add thisto the other section as well

**Comment by awanlin** [comment]
@alefcarlos awesome, just something to note I'm not sure if you can legally use the .Net logo. We couldn't use the logos for the Azure DevOps plugin we created.

File: microsite/data/plugins/scaffolder-backend-dotnet.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: Scaffolder Backend .NET
+author: Alef Carlos
+authorUrl: https://github.com/alefcarlos
+category: Scaffolder
+description: Here you can find all .NET related actions to improve your scaffolder.
+documentation: https://github.com/alefcarlos/plusultra-dotnet-backstage-plugins/blob/main/plugins/scaffolder-dotnet-backend/README.md
+iconUrl: img/scaffolder-backend-dotnet-icon.png
+npmPackageName: '@plusultra/plugin-scaffolder-dotnet-backend'
```
Comment (awanlin) [inline microsite/data/plugins/scaffolder-backend-dotnet.yaml:9]: You'll need a new line here at the end. The builds when they run will fail on this

**Review by awanlin — COMMENTED** [review]
Glad to see some .NET "action" going on 😉 

Just one comment and then as we previously chatted about the image will need to be changed

**Comment by alefcarlos** [comment]
@awanlin Im gonna send you a new action on your Azure Pipeline plugin, I created a custom action to create Pipeline Build Defition on Azure, but it still has some implementation detail that I have to adjust yet. I really enjoyed you awesome job!

File: microsite/data/plugins/scaffolder-backend-dotnet.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+title: Scaffolder Backend .NET
```
Comment (Rugvip) [inline microsite/data/plugins/scaffolder-backend-dotnet.yaml:2]: What do you think about calling it `Scaffolder .NET Actions`? Just to provide a bit more context

**Review by Rugvip — COMMENTED** [review]
Thanks! 👍

File: microsite/data/plugins/scaffolder-backend-dotnet.yaml (hunk 2 above)
Comment (Rugvip) [reply microsite/data/plugins/scaffolder-backend-dotnet.yaml:2]: Hi! What do you think about this suggestion?

**Comment by alefcarlos** [comment]
@Rugvip I though I had changed the name, but I forget hahahaha

Now its okay!

**Comment by alefcarlos** [comment]
@Rugvip Hi, a have 1 failing check because of my own Name hahahaha,

image

How could I add my name on a whitelist?

**Comment by manusant** [comment]
> @Rugvip Hi, a have 1 failing check because of my own Name hahahaha,
> 
> image
> 
> How could I add my name on a whitelist?

Hi @alefcarlos you can do it here: https://github.com/backstage/backstage/blob/master/.github/styles/vocab.txt

**Review by Rugvip — APPROVED** [review]
Thanks! 😁 

:shipit:

**Pull request merged**